src/pheval/config_parser.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+import yaml
+from serde import serde
+from serde.yaml import from_yaml
+
+
+@serde
```

Comment:
What does this do?

Reply:
This has been the library I'm using to parse any yaml config (I started using it in the Exomiser runner). So it will parse the contents into the dataclass so I would be able to access elements like so: ```input_dir_config.tool```, while before I would be accessing it as a dictionary element ```input_dir_config["tool"]```.

It will work if there are extra fields in the config that are not documented in the dataclass but I'm not exactly sure if these can be accessed if not specified in the dataclass

Reply:
Cool!

Reply:
cc @souzadevinicius just FYI, this is something interesting I just learned and never used before!